### ISSUE TITLE
Question: Can variables be XORed directly when using biguint?

### ISSUE THREAD

**Issue by phantomcraft** [SzigetiJ/biguint#32]
Like this:

```
BigUInt256 var1 = 0xf5109583c8c61e884a22ed50fd579ac519ca46829d7f4c03aee883a02f6e0793;
BigUInt256 var2 = 0xc96ed6f80c1a691d0373772e531d5ee4aaeca263f4901423b27177e5f18dd2cf;

var1 ^= var2;
```

**Comment by SzigetiJ** [comment]
XOR-assignment is not imlemented yet.

**Comment by SzigetiJ** [comment]
XOR-assigment is now implemented: #36 along with functions realizing `&=`, `|=` and `a=~a` operators:
```c
BigUInt128 *biguint128_and_assign(BigUInt128 *a, const BigUInt128 *b);
BigUInt128 *biguint128_or_assign(BigUInt128 *a, const BigUInt128 *b);
BigUInt128 *biguint128_not_assign(BigUInt128 *a);
BigUInt128 *biguint128_xor_assign(BigUInt128 *a, const BigUInt128 *b);
```
